Ports/kakoune/package.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='kakoune'
+version='674053935d3260dd54c4e0084c76e9170d731eb2'
```

Comment:
This looks outdated by now, maybe we'll want to bump that to the latest commit before shipping.